### PR TITLE
Carcas done.

### PR DESCRIPTION
**Pull request by NickiHell**

### CODE REVIEW

**Comment by NickiHell** [comment]
??? @de10rean

**Comment by NickiHell** [comment]
#1 Starting up!

File: wwwm/settings.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'WWWM.apps.WwwmConfig',
+    'diagnosis.apps.WwwmConfig',
```
Comment (de10rean) [inline wwwm/settings.py]: You can import "diagnosis" only it will be enough and will be more correct

File: diagnosis/apps.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class WwwmConfig(AppConfig):
```
Comment (de10rean) [inline diagnosis/apps.py]: Fix class name

File: wwwm/urls.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+def index(request):
+    return HttpResponse("MAIN PAGE!!")
+
+def dimon(request):
+    return HttpResponse("ДИМОН НАХУЙ ПРИМИ ПУЛ!")
+
+urlpatterns = [
+    path('', index),
+    path('dimon/', dimon),
```
Comment (de10rean) [inline wwwm/urls.py]: Take it off...yomayo

File: wwwm/urls.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
+from django.http import HttpResponse
+
+def index(request):
```
Comment (de10rean) [inline wwwm/urls.py]: Writing views in urls.py is criminal. I call the police. Take it of

**Comment by NickiHell** [comment]
UKRAINE PROJECT!

File: diagnosis/apps.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class diagnosisConfig(AppConfig):
```
Comment (de10rean) [inline diagnosis/apps.py]: PEP8 said that class name must be in capitalized CamleCase style

File: wwwm/settings.py [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+INSTALLED_APPS = [
+    'bootstrap3',
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'diagnosis.apps.DiagnosisConfig',
```
Comment (de10rean) [inline wwwm/settings.py]: Just diagnosis

**Pull request merged**